localstack/setup-sns.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
file only required to run locally with docker-compose and is taken from court-register